controllers/backplaneconfig_controller.go

```diff
@@ -517,7 +517,8 @@ func (r *MultiClusterEngineReconciler) ensureToggleableComponents(ctx context.Co
 	errs := map[string]error{}
 	requeue := false
 
-	if backplaneConfig.Enabled(backplanev1.ManagedServiceAccount) {
+	if backplaneConfig.Enabled(backplanev1.ManagedServiceAccount) ||
+		backplaneConfig.Enabled(backplanev1.ManagedServiceAccountPreview) {
```

Comment:
This is still taking into account the preview's configuration, which doesn't align with the criteria
> managedserviceaccount-preview component still recognized, but ignored.